packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -20,14 +20,17 @@ class DragDownDetails {
   /// Creates details for a [GestureDragDownCallback].
   ///
   /// The [globalPosition] argument must not be null.
-  DragDownDetails({ this.globalPosition = Offset.zero })
+  DragDownDetails({ this.globalPosition = Offset.zero, this.deviceKind = PointerDeviceKind.touch})
```

Comment:
nit: please make sure to be consistent in the style. so for example here, you have a space after the `{` so you should have a space before the `}`.